WordPress/src/main/java/org/wordpress/android/ui/comments/unified/UnifiedCommentsEditViewModel.kt

```diff
@@ -285,6 +295,26 @@ class UnifiedCommentsEditViewModel @Inject constructor(
         }
     }
 
+    private suspend fun updateReaderEntity(commentEssentials: CommentEssentials) {
+        val readerCommentIdentifier = commentIdentifier as ReaderCommentIdentifier
+
+        val readerComment = readerCommentTableWrapper.getComment(
+                site.siteId,
+                readerCommentIdentifier.postId,
+                readerCommentIdentifier.remoteCommentId
+        )
+
+        readerComment?.let {
```

Comment:
Nitpick: do you think it would make sense to use `apply` instead?

e.g.
```
        readerComment?.apply {
            text = commentEssentials.commentText
            authorName = commentEssentials.userName
            authorEmail = commentEssentials.userEmail
            authorUrl = commentEssentials.userUrl
            readerCommentTableWrapper.addOrUpdateComment(readerComment)
        }
```

Reply:
Yep, apply definitely makes more sense here :) Fixed it!